src/analyzer/tests/cookies.js

```diff
@@ -94,9 +94,8 @@ export function cookiesTest(
   }
 
   // get ALL the cookies from the store with serializeSync instead of using getCookiesSync
-  const allCookies = requests.session?.jar
-    .serializeSync()
-    .cookies.filter(filterCookies);
+  const allCookies =
+    requests.session?.jar?.serializeSync()?.cookies.filter(filterCookies) ?? [];
 
   if (!allCookies?.length) {
```

Comment:
Nit: Isn't `allCookies` guaranteed to be an array now?
```suggestion
  if (!allCookies.length) {
```